fluent-bit.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - flex
+      - glibc
+      - openssl-dev
+      - yaml-dev
+      - zlib-dev
```

Comment:
If you want the systemd/journald input plugin then we need to add the `libsystemd-dev` or equivalent.
It's useful to have for kubelet logs or other systemd services: https://docs.fluentbit.io/manual/pipeline/inputs/systemd

Reply:
The `-dev` is not but the libraries are - same for the others. We use `cpack` to handle it currently, i.e. auto-require things.